TESTS/mbed_platform/minimal-printf/compliance/main.cpp

```diff
@@ -42,6 +42,94 @@
 
 using namespace utest::v1;
 
+#if defined(__NEWLIB_NANO)
+
+#define MAX_STRING_SIZE 100
+//Global buffer used by test cases to store string literal
+char expected_string[MAX_STRING_SIZE];
+
+typedef enum {
+    RADIX_10,//radix base for decimal
+    RADIX_16 //radix base for Hexadecimal
+} radix_base;
```

Comment:
Enum names normally use the CapWords convention in Mbed OS.
```suggestion
} BaseType;
```
Radix and Base denote the same thing so `RadixBase` is not a good name. Why don't you just use the commonly use term `base` like `itoa()` does?